lib/packages/docutils/rst.nim

```diff
@@ -277,6 +277,7 @@ type
     line*, col*, baseIndent*: int
     skipPounds*: bool
     adornmentLine*: bool
+    escapeNext*: bool
```

Comment:
Interactions between the lexer and the parser via different lexing states is a most ugly setup. I understand why you did it this way and as you said, it's likely the cleanest solution, but these special cases then live in the code *for good*. Which is why I'm now musing over moving to Markdown instead.

Reply:
> but these special cases then live in the code for good

I don't know, I'm inclined to think it's OK and it should work this way, because ``\`` **does** have the special escaping meaning *for good* — in both RST and Markdown.